File: 27-RemoveElement/javascript/solution.js
// Last updated: 3/6/2026, 12:46:24 AM
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function (nums, val) {
    let i = 0;
    for (let int of nums) {
        if (int != val) {
            nums[i] = int;
            i++;
        }
    }
    return i
};

File: 27-RemoveElement/cpp/solution.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int i = 0;
        for (int check : nums) {
            if (check != val) {
                nums[i] = check;
                i++;
            }
        }
        return i;
    }
};